check-commit-msg.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+
+const COMMIT_RULE = /^(build|ci|chore|docs|feat|fix|perf|refactor|style|test)(\\([a-z\\-]+\\))?: .*/;
+
+const commitMessageFilename = process.env.GIT_PARAMS; // HUSKY_GIT_PARAMS for husky@next
+const commitMessage = fs.readFileSync(commitMessageFilename, 'utf8');
+
+const compliance = commitMessage.match(COMMIT_RULE);
+
+if (!compliance) {
+  console.log('The commit message does not comply with the contributing rules!');
```

Comment:
I would suggest "The commit message does not comply with the contributing rules. See Contributing.md."